### PR TITLE
Fix 404 handling for not found posts.

### PR DESCRIPTION
**Pull request by LukeTowers**
Follow up to #438. Refs: https://octobercms.com/forum/post/error-when-trying-to-navigate-away-from-blog-page

### CODE REVIEW

File: components/Post.php [hunk 1]
```diff
@@ -84,27 +87,25 @@ protected function loadPost()
     {
         $slug = $this->property('slug');
 
-        $post = new BlogPost;
-
-        $post = $post->isClassExtendedWith('RainLab.Translate.Behaviors.TranslatableModel')
-            ? $post->transWhere('slug', $slug)
-            : $post->where('slug', $slug);
+        $post = BlogPost;
```
Comment (bennothommo) [inline components/Post.php:90]: @LukeTowers I've never seen this syntax before - does this still instantiate a `BlogPost` instance?

Comment (LukeTowers) [reply components/Post.php:90]: @bennothommo nope 🤦 That's just called "Luke shouldn't be writing code at 1:30am" 😂

Comment (LukeTowers) [inline components/Post.php:90]:
```suggestion
        $post = new BlogPost;
```

**Comment by bennothommo** [comment]
Tested and confirmed working.

**Pull request merged**